### PR TITLE
games-engines/qtads-3.0.0: version bump, delete old

### PR DESCRIPTION
**Pull request by realnc**
The new version gets rid of the need for "-fpermissive" and now also provides a make install target. The "sound" global USE flag has been added as well, since it's now possible to build without sound support.

Also updated metadata (added description and new upstream location as it moved from sourceforge to github.)

Bumped EAPI to 7.

### CODE REVIEW

File: games-engines/qtads/qtads-3.0.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+KEYWORDS="~amd64 ~x86"
+IUSE="+sound"
+
+DEPEND="dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtwidgets:5
+	sound? (
+		media-libs/libsdl2[sound]
+		media-libs/libsndfile
+		media-libs/libvorbis
+		media-sound/fluidsynth
```
Comment (chewi) [inline games-engines/qtads/qtads-3.0.0.ebuild]: `media-sound/fluidsynth` has a subslot now so this should include `:0=`.

File: games-engines/qtads/qtads-3.0.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+HOMEPAGE="https://realnc.github.io/qtads"
+SRC_URI="https://github.com/realnc/qtads/releases/download/v${PV}/${P}-source.tar.xz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+sound"
+
+DEPEND="dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtwidgets:5
```
Comment (chewi) [inline games-engines/qtads/qtads-3.0.0.ebuild]: I know it's somewhat implied but I think you should also have `dev-qt/qtcore` and `dev-qt/qtgui` here as they explicitly appear in the final link and `-Wl,--as-needed` doesn't prune them from the `NEEDED` entries.

**Comment by realnc** [comment]
> Good stuff, just a couple of fixes needed.

Thanks. Fixed.

**Comment by realnc** [comment]
I forgot that there's `BDEPEND` as well now with EAPI 7, and this package needs pkg-config. Added it as well.

**Pull request closed without merging**